AutomatedTesting/Gem/PythonTests/scripting/scripting_utils/scripting_constants.py

```diff
@@ -71,6 +71,7 @@
 SAVE_STRING = "Save"
 NAME_STRING = "Name"
 WAIT_FRAMES = 200
+WAIT_TIME_1 = 1
 WAIT_TIME_3 = 3
 WAIT_TIME_5 = 5
```

Comment:
Backlog: We should deprecate the timing scripting constants for the timing constants in `AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/consts/general.py`

Reply:
i agree. something this generic seems like it shouldn't live in script canvas land. we should talk about planning something like this. It may have to way until after holiday season though